elephant/spike_train_correlation.py

```diff
@@ -690,8 +690,17 @@ def run_T(spiketrain, N, dt):
         PA = PA / N1
         PB = run_P(spiketrain_2, spiketrain_1, N2, N1, dt)
         PB = PB / N2
-        index = 0.5 * (PA - TB) / (1 - PA * TB) + 0.5 * (PB - TA) / (
-            1 - PB * TA)
+        # check if the P and T values are 1 to avoid division by zero
+        if PA * TB == 1:
+            if PB * TA == 1:
+                index = 1.
```

Comment:
Setting `index == 1` should be documented. Also why 1?

Reply:
I added an explanation in the documentation and in the comment next to the if clauses. In those cases, every spike in the train with P = 1 is within dt of a spike in the other train, so I set the respective (partial) index to 1.

Reply:
Is it possible to add a test for the 0/0 case? I guess you had such a case?